sdk/chainpatrol-client/instance-methods/asset-list.mdx

```diff
@@ -1,99 +1,424 @@
+</Card>
+
+  <Card title="By Organization" icon="building">
+    Use `organizationSlug` to get organization-specific results.
+  </Card>
+</CardGroup>
+
+## Getting Started
+
+### Quick Start
+
+<CodeGroup>
+```typescript Basic List Query
+import { ChainPatrolClient } from "@chainpatrol/sdk";
+
+const client = new ChainPatrolClient({
+apiKey: "cp_live_your_api_key_here",
+});
+
+const result = await client.asset.list({
+type: "URL",
+status: "BLOCKED",
+startDate: "2023-01-01",
+endDate: "2023-12-31",
+});
+
+console.log(`Found ${result.assets.length} blocked URLs`);
+
+````
+
+```typescript Organization-Specific Query
+const result = await client.asset.list({
+  type: "ADDRESS",
+  status: "BLOCKED",
+  startDate: "2024-01-01",
+  endDate: "2024-01-31",
+  organizationSlug: "ethereum-foundation",
+});
+
+result.assets.forEach(asset => {
+  console.log(`Blocked address: ${asset.content}`);
+  console.log(`Updated: ${asset.updatedAt}`);
+});
+````
+
+```json Example Response
+{
+  "assets": [
+    {
+      "type": "URL",
+      "content": "phishing-site.com",
+      "status": "BLOCKED",
+      "updatedAt": "2023-06-15T10:30:00.000Z",
+      "reason": "reported"
+    },
+    {
+      "type": "URL",
+      "content": "scam-airdrop.net",
+      "status": "BLOCKED",
+      "updatedAt": "2023-06-14T15:45:00.000Z",
+      "reason": "eth-phishing-detect"
+    }
+  ]
+}
+```
+
+</CodeGroup>
+
+### Basic Usage Patterns
+
+```typescript
+// Get recent threats
+async function getRecentThreats(days: number = 7) {
+  const endDate = new Date().toISOString().split("T")[0];
+  const startDate = new Date(Date.now() - days * 24 * 60 * 60 * 1000)
+    .toISOString()
+    .split("T")[0];
+
+  return await client.asset.list({
+    type: "URL",
+    status: "BLOCKED",
+    startDate,
+    endDate,
+  });
+}
+
+// Count assets by status
+async function getAssetStats(dateRange: { start: string; end: string }) {
+  const [blocked, allowed] = await Promise.all([
+    client.asset.list({
+      type: "URL",
+      status: "BLOCKED",
+      startDate: dateRange.start,
+      endDate: dateRange.end,
+    }),
+    client.asset.list({
+      type: "URL",
+      status: "ALLOWED",
+      startDate: dateRange.start,
+      endDate: dateRange.end,
+    }),
+  ]);
+
+  return {
+    blocked: blocked.assets.length,
+    allowed: allowed.assets.length,
+  };
+}
+```
+
+## Implementation Guide
+
+### Use Cases & Patterns
+
+<AccordionGroup>
+  <Accordion title="Threat Intelligence Reports">
+    Generate periodic reports on blocked assets for security analysis.
+    
+    ```typescript
+    async function generateThreatReport(month: string) {
+      const startDate = `${month}-01`;
+      const endDate = `${month}-31`;
+      
+      const threats = await client.asset.list({
+        type: "URL",
+        status: "BLOCKED",
+        startDate,
+        endDate,
+      });
+      
+      // Group by reason
+      const byReason = threats.assets.reduce((acc, asset) => {
```

Comment:
The "Reason" string is actually not really used anymore, we should mark reason as deprecated and not give examples with it